compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -114,149 +114,101 @@ struct NoOpDetector final : public luci::CircleNodeMutableVisitor<bool>
   bool visit(luci::CircleNode *) final { return false; }
 };
 
-void allocateCircleTensorInfo(CircleNode *node, CircleTensorContext &ctx)
+class CircleTensorInfoAllocationRule final : public luci::CircleNodeMutableVisitor<void>
```

Comment:
I don't agree on introducing `CircleTensorInfoAllocationRule` to mix tensor allocation and multioutput detection.

Reply:
It is not mixing :)
It is just allocating, not detecting.